src/OperationResponse.php

```diff
@@ -241,9 +299,9 @@ public function getResult()
      * @return Status|null The status of the operation in case of failure, or null if
      *                                 operationFailed() is false.
      */
-    public function getError()
+    public function getError(Message $lastProtoResponse = null)
     {
-        if (!$this->isDone() || is_null($this->lastProtoResponse->getError())) {
+        if (!$this->isDone()) {
             return null;
         }
         return $this->lastProtoResponse->getError();
```

Comment:
Just want to double check: if `$lastProtoResponse` is in fact `null`, does this throw an exception? I guess this case isn't possible because `$this->isDone()` would return `false` if `$this->lastProtoResponse` was `null`.

Reply:
That's correct. But for extra readability and safety I'll add a call to `$this->hasProtoResponse()` here also.

Reply:
SGTM. Thanks.